docs/quick-start/transmitters/lua-howto.md

```diff
@@ -168,46 +205,4 @@ The Other Devices folder, if present, allows changing the configuration of other
 
 * For freestyle and general everyday flying, you may use the `250Hz` modes or faster, with the `Std` Telemetry mode. Switch Mode doesn't matter as much here. Select what's appropriate for your Flight Mode settings.
 
-* For fixed wings, we recommend using `100Hz Full Res`, together with either `Std` or your choice of Telemetry Ratio. Switch Mode will depend greatly on how many full resolution channels you intend to use.
-
-## Troubleshooting the Lua Script
```

Comment:
Why is this section being removed?

Yes, it can go into Troubleshooting Page, which users don't usually visit.
So it is OK there's duplication of these section where appropriate. 

IMHO

Reply:
Yep, you are right. I will just duplicate it on the troubleshooting page.